MediaBrowser.Providers/MediaInfo/VideoImageProvider.cs

```diff
@@ -78,12 +83,15 @@ private async Task<DynamicImageResponse> GetVideoImage(Video item, CancellationT
 
             // If we know the duration, grab it from 10% into the video. Otherwise just 10 seconds in.
             // Always use 10 seconds for dvd because our duration could be out of whack
-            var imageOffset = item.VideoType != VideoType.Dvd && item.RunTimeTicks.HasValue &&
-                              item.RunTimeTicks.Value > 0
+            var imageOffset = item.VideoType != VideoType.Dvd && item.RunTimeTicks is > 0
                                   ? TimeSpan.FromTicks(item.RunTimeTicks.Value / 10)
                                   : TimeSpan.FromSeconds(10);
 
-            var videoStream = item.GetDefaultVideoStream() ?? item.GetMediaStreams().FirstOrDefault(i => i.Type == MediaStreamType.Video);
+            var defaultQuery = new MediaStreamQuery { ItemId = item.Id, Index = item.DefaultVideoStreamIndex };
+            var videoQuery = new MediaStreamQuery { ItemId = item.Id, Type = MediaStreamType.Video };
+
+            var videoStream = _mediaSourceManager.GetMediaStreams(defaultQuery).FirstOrDefault()
+                ?? _mediaSourceManager.GetMediaStreams(videoQuery).FirstOrDefault();
```

Comment:
It's a wasted allocation if the first call doesn't return null
```suggestion
            var query = new MediaStreamQuery { ItemId = item.Id, Index = item.DefaultVideoStreamIndex };
            var videoStream = _mediaSourceManager.GetMediaStreams(defaultQuery).FirstOrDefault();
            if (videoStream == null)
            {
                videoQuery.Type = MediaStreamType.Video;
                videoQuery.Index = null;
                videoStream = _mediaSourceManager.GetMediaStreams(query).FirstOrDefault();
            }
```